README.md

```diff
@@ -37,6 +42,38 @@ Instead of specifying the packages directly, you can pass a file containing the
            package_file: tl_packages
 ```
 
+### Enabling updating cache using a workflow trigger
+
+One can inclue a cache referesh using GitHub's `workflow_dispatch` functionality:
+
+```yaml
+name: check
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      cache_version:
+        description: 'Cache version for install-texlive'
+        default: '0'
```

Comment:
If I see this correctly this requires the user to manually specify the current `cache_version`. This seems rather error-prone since specifying anything else than the one used in the actual main workflow would generate a cache which is completely ignored afterwards.

I think I'm failing to see the use-case here. If this is used it doesn't seem to use anything which isn't done anyway by a regular run.

Reply:
Maybe, it's to edge casy: My thing is, that I have a block of time for texlatex now and then - and not on a regular basis. Last time, TeXLive was released I needed to refresh the cache of all install-texlive-based workflows. Later that week, a refresh of newtxtt was required. I think, the latter one is automatically supported by install-texlive, the former not. Since this is a once-a-year thing, maybe, it is only for pro-users...

I'll close and hope, there will be a fix for automatic TL 2023 to TL 2024 updates. Refs https://github.com/zauguin/install-texlive/issues/16.